src/test/java/bio/terra/workspace/service/spendprofile/SpendProfileServiceTest.java

```diff
@@ -0,0 +1,91 @@
+package bio.terra.workspace.service.spendprofile;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import bio.terra.workspace.app.configuration.external.SpendProfileConfiguration;
+import bio.terra.workspace.common.BaseUnitTest;
+import bio.terra.workspace.common.utils.SamUtils;
+import bio.terra.workspace.service.iam.SamService;
+import bio.terra.workspace.service.spendprofile.exceptions.SpendUnauthorizedException;
+import com.google.common.collect.ImmutableList;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.mockito.Mockito;
+import org.springframework.boot.test.mock.mockito.MockBean;
+
+// TODO(PF-186): Consider if this should be a connected test that talks to Sam.
+public class SpendProfileServiceTest extends BaseUnitTest {
+  @MockBean SamService mockSamService;
+
+  /** Fake user access token. */
+  private static final String USER_ACCESS_TOKEN = "fake-token";
+
+  @BeforeEach
+  public void setUp() {
+    // By default, allow the user access token to link spend profile resources for any id.
+    Mockito.when(
+            mockSamService.isAuthorized(
+                Mockito.eq(USER_ACCESS_TOKEN),
+                Mockito.eq(SamUtils.SPEND_PROFILE_RESOURCE),
+                Mockito.any(),
+                Mockito.eq(SamUtils.SPEND_PROFILE_LINK_ACTION)))
+        .thenReturn(true);
+  }
+
+  @Test
+  public void authorizeLinking_Success() {
```

Comment:
nit, afaik other tests in WM just use camelcase without underscores for names. I'd vote for consistency here, but it's not a strong opinion

Reply:
I've already added underscores in both WorkspaceDaoTest & MdcHookTest. But I also don't feel strongly.